packages/pf4-component-mapper/src/form-fields/wizard/wizard.js

```diff
@@ -19,7 +21,9 @@ class Wizard extends React.Component {
     super(props);
 
     // find if wizard contains any dynamic steps (nextStep is mapper object)
-    const isDynamic = this.props.isDynamic ? true : this.props.fields.find(({ nextStep }) => typeof nextStep === 'object') ? true : false;
+    const isDynamic = this.props.isDynamic ? true : this.props.fields.find(({ nextStep }) => DYNAMIC_WIZARD_TYPES.includes(typeof nextStep))
+      ? true
```

Comment:
:1st_place_medal: This reminds us some code from ManageIQ. I dont think we need ternary operator here. Maybe use `!!find()...`

Reply:
:face_with_thermometer: